### DISCUSSION TITLE
打开后闪退

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [cangzhang/ChampR#271] [General]
最新版本2.0.2b2，画面一闪而过，之后软件闪退
![image](https://github.com/cangzhang/champ-r/assets/42333702/4995a01a-7d90-4393-b80f-8c3c8cfceadb)

**Comment by cangzhang** [comment]
先打开 lol，再打开 ChampR

**Reply by zhaoliang0302** [reply]
一样的闪退

**Reply by cangzhang** [reply]
系统是官方系统吗？
本地网络可以访问 https://ql.lbj.moe 吗？

**Reply by zhaoliang0302** [reply]
这个网址进去报错：Application error: a client-side exception has occurred (see the browser console for more information).
Digest: 4266806755

**Reply by cangzhang** [reply]
报错是正常的

**Reply by cangzhang** [reply]
可以试试新版本 https://github.com/cangzhang/champ-r/releases/tag/v2.0.2-b3
